### ISSUE TITLE
Magma Server doesn't start with the newest version of MorePlanets 

### ISSUE THREAD

**Issue by cestofmir** [SteveKunG/MorePlanets#160]
**Server log**: https://pastebin.com/5baSHqzh

Using the latest version of MorePlanets, my Magma server isn't starting up. Without the MorePlanets mod, the server seems to startup just fine. Nothing seems to show up in the server console, all it says is "A problem occurred running the Server launcher."

I have a suspicion it has something to do with the "Survival Planet" feature.

**Comment by SteveKunG** [comment]
I don't see any log that related to MP, did you put the right one?

**Comment by KingColton1** [comment]
I started using Magma few days ago but I ran into problem. After multiple testing with Magma, I found out that MorePlanets caused to stop from running Magma server. I'm experiencing same issue as OP so here's a full server log: https://pastebin.com/WgT8r1vP. The bottom of the log always show same error coming from MorePlanets every time I run and fail.

Here's a specific line that caused to stop from running server:
`[17:07:25] [main/INFO] [STDERR]: [java.lang.Throwable:printEnclosedStackTrace:710]: Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Constant modifier method moreplanets$changeDim(I)I in mixins.more_planets.json:MixinEntity failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap mixins.more_planets.refmap.json`

When I remove MorePlanets mod from Magma server, it run just fine with all of other mods.
